### DISCUSSION TITLE
Pocketbase Gopher

### DISCUSSION THREAD

**Discussion by calumk** [pocketbase/pocketbase#807] [Q&A]
1) Does it have a name?
2) Is it just the go gopher, or is it a cousin or a relative?

**Comment by ganigeorgiev** [comment]
It's just a go gopher :)

It's a slightly modified vector of one of the gophers by @marcusolsson from https://github.com/marcusolsson/gophers (_there is a credit to the repo in the html if you inspect the image in pocketbase.io_).

**Reply by marcusolsson** [reply]
Thanks for the credits 👋  Would it be possible to also add a tooltip with the credit (for people who don't think to open the Developer Console)?

**Reply by ganigeorgiev** [reply]
@marcusolsson Yep, I will add a tooltip with the v0.8 release (_or maybe directly a link in the footer_).
